test/manual/amp-pinterest.amp.html

```diff
@@ -0,0 +1,97 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Pinterest examples</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script custom-element="amp-pinterest" src="../dist/v0/amp-pinterest-0.1.max.js" async></script>
+  <script src="../dist/amp.js" async ></script>
+  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+</head>
+<body>
+
+  <h2>Pinterest Buttons and Widgets</h2>
+
+  <h3>Pin It Buttons</h3>
+
+  <h4>Small Gray</h4>
+
+    <amp-pinterest height=20 width=40
+      data-do="buttonPin"
+      data-url="http://www.flickr.com/photos/kentbrew/6851755809/"
+      data-media="http://c2.staticflickr.com/8/7027/6851755809_df5b2051c9_b.jpg"
+      data-description="Next stop: Pinterest">
+    </amp-pinterest>
+
+  <h4>Large Circle</h4>
+
+    <amp-pinterest height=32 width=32
+      data-do="buttonPin"
+      data-shape="round"
+      data-height="32"
```

Comment:
Sorry, my fault! Forgot to update this file.

For the Pin It button, `data-height=somenumber` has been deprecated in favor of `data-tall="true"` because we know how tall a particular button needs to be based on shape. (Which has also been changed from `data-shape="round"` to `data-round="true"`.) Make sense?